README.md

```diff
@@ -2,28 +2,28 @@
 
 ![Amazon ECS logo](doc/ecs.png "Amazon ECS")
 
-![Build Status](https://github.com/aws/amazon-ecs-agent/workflows/Build/badge.svg?branch=dev)
-
 The Amazon ECS Container Agent is a component of Amazon Elastic Container Service
 ([Amazon ECS](http://aws.amazon.com/ecs/)) and is responsible for managing containers on behalf of Amazon ECS.
 
+This repository comes with ECS-Init, which is a [systemd](http://www.freedesktop.org/wiki/Software/systemd/) based service to support the Amazon ECS Container Agent and keep it running. It is used for systems that utilize `systemd` as init systems and is packaged as deb or rpm. The source for ECS-Init is available in this repository at `./ecs-init` while the packaging is available at `./packaging`.
+
 ## Usage
 
 The best source of information on running this software is the
 [Amazon ECS documentation](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_agent.html).
 
-Please note that from Agent version 1.20.0, Minimum required Docker version is 1.9.0, corresponding to Docker API version 1.21. For more information, please visit [Amazon ECS Container Agent Versions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container_agent_versions.html).
-
 ### On the Amazon Linux AMI
 
 On the [Amazon Linux AMI](https://aws.amazon.com/amazon-linux-ami/), we provide an installable RPM which can be used via
 `sudo yum install ecs-init && sudo start ecs`. This is the recommended way to run it in this environment.
 
 ### On Other Linux AMIs
 
+[Amazon ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-install.html) provides deb and rpm packages and instructions to install ECS Container Agent on non-Amazon Linux instances.
+
 The Amazon ECS Container Agent may also be run in a Docker container on an EC2 instance with a recent Docker version
 installed. A Docker image is available in our
-[Docker Hub Repository](https://registry.hub.docker.com/u/amazon/amazon-ecs-agent/).
+[Docker Hub Repository](https://hub.docker.com/r/amazon/amazon-ecs-agent).
```

Comment:
Non-blocking; should we also put the ECR public repo link here? 

https://gallery.ecr.aws/ecs/amazon-ecs-agent

Reply:
Thank you for noticing this, should be good to add this

Reply:
Included in #3555